mypy/fixup.py

```diff
@@ -312,8 +298,7 @@ def missing_info(modules: Dict[str, MypyFile]) -> TypeInfo:
     dummy_def.fullname = suggestion
 
     info = TypeInfo(SymbolTable(), dummy_def, "<missing>")
-    obj_type = lookup_qualified(modules, 'builtins.object', False)
-    assert isinstance(obj_type, TypeInfo)
+    obj_type = lookup_qualified_typeinfo(modules, 'builtins.object', False)
```

Comment:
For bool arguments it will be clearer to use a keyword argument instead of positional.